worldeditadditions_commands/commands/selectors/sshift.lua

```diff
@@ -8,46 +7,26 @@ local Vector3 = worldeditadditions.Vector3
 --      ██      ██ ██   ██ ██ ██         ██
 -- ███████ ███████ ██   ██ ██ ██         ██
 
-local function parse_with_name(name,args)
-	local vec, tmp = Vector3.new(0, 0, 0), {}
-	local find, _, i = {}, 0, 0
-	repeat
-		_, i, tmp.proc = args:find("([%l%s+-]+%d+)%s*", i)
-		if tmp.proc:match("[xyz]") then
-			tmp.ax = tmp.proc:match("[xyz]")
-			tmp.dir = tonumber(tmp.proc:match("[+-]?%d+")) * (tmp.proc:match("-%l+") and -1 or 1)
-		else
-			tmp.ax, _ = wea_c.dir_to_xyz(name, tmp.proc:match("%l+"))
-			if not tmp.ax then return false, _ end
-			tmp.dir = tonumber(tmp.proc:match("[+-]?%d+")) * (tmp.proc:match("-%l+") and -1 or 1) * _
-		end
-		vec[tmp.ax] = tmp.dir
-	until not args:find("([%l%s+-]+%d+)%s*", i)
-	return true, vec
-end
-
 worldeditadditions_core.register_command("sshift", {
-	params = "<axis1> <distance1> [<axis2> <distance2> [<axis3> <distance3>]]",
+	params = "<unified axis syntax>",
 	description = "Shift the WorldEdit region in 3 dimensions.",
 	privs = { worldedit = true },
 	require_pos = 2,
 	parse = function(params_text)
-		if params_text:match("([%l%s+-]+%d+)") then return true, params_text
-		else return false, "No acceptable params found" end
+		local ret = wea_c.split(params_text)
+		if #ret < 1 then return false, "SSHIFT: No params found!"
```

Comment:
Perhaps we can make this a little more user-friendly?
```suggestion
		if #ret < 1 then return false, "Error: No params found!"
```
.....given that the user is executing `//sshift` (or an alias thereof), then it should be obvious where the error comes from?